monai/apps/pathology/handlers.py

```diff
@@ -0,0 +1,99 @@
+import logging
+import os
+from typing import TYPE_CHECKING, Dict, Optional
+
+import numpy as np
+
+from monai.utils import exact_version, optional_import
+
+Events, _ = optional_import("ignite.engine", "0.4.4", exact_version, "Events")
+if TYPE_CHECKING:
+    from ignite.engine import Engine
+else:
+    Engine, _ = optional_import("ignite.engine", "0.4.4", exact_version, "Engine")
+
+
+class ProbMapGenerator:
+    """
+    Event handler triggered on completing every iteration to save the probability map
+    """
+
+    def __init__(
+        self,
+        output_dir: str = "./",
+        name: Optional[str] = None,
```

Comment:
shall we have a `dtype` arg for the saved data precision? and `output_postfix` like https://github.com/Project-MONAI/MONAI/blob/43f482c4ace9cbbc1da50d994edf389ec17f1096/monai/handlers/segmentation_saver.py#L53